.mvn/develocity.xml

```diff
@@ -0,0 +1,30 @@
+<develocity
+		xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
+	<server>
+		<url>https://ge.spring.io</url>
+	</server>
+	<buildScan>
+		<backgroundBuildScanUpload>false</backgroundBuildScanUpload>
+		<captureGoalInputFiles>true</captureGoalInputFiles>
+		<publishIfAuthenticated>true</publishIfAuthenticated>
+		<obfuscation>
+			<ipAddresses>#{{'0.0.0.0'}}</ipAddresses>
+		</obfuscation>
+	</buildScan>
```

Comment:
Do we want to have this on our CI Builds or also on releases?   If so do we want to use tags?
```
    <tags>
      <tag>my tag</tag>
    </tags>
```